wallet/reference/multichain-api.md

```diff
@@ -383,4 +383,18 @@ according to [CAIP-217](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/c
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Error codes
+
+The Multichain API can return the following JSON-RPC error codes:
+
+| Code | Description | Related methods |
+| ---- | ----------- | --------------- |
+| 5000 | Unknown error with request | `wallet_revokeSession`, `wallet_createSession` | 
+| 5100 | Requested networks are not supported | `wallet_createSession` |
+| 5101 | Requested methods are not supported | `wallet_createSession` |
+| 5102 | Requested notifications are not supported | `wallet_createSession` |
+| 5300 | Invalid `scopedProperties` requested | `wallet_createSession`|
+| 5301 | `scopedProperties` can only be outside of `sessionScopes` | `wallet_createSession`|
```

Comment:
i believe we treat this as invalid scope object and simply ignore the scopeobject. thoughts, @adonesky1 ?